src/components/calcite-pick-list-item/calcite-pick-list-item.tsx

```diff
@@ -217,6 +221,10 @@ export class CalcitePickListItem {
     }
   };
 
+  contentSlotClickHandler = (event: MouseEvent): void => {
+    event.stopPropagation();
```

Comment:
What is the reasoning for stopping propagation here?

Reply:
When label/description text content is clicked, the item container gets focused. In this case when the slot has input and when input is clicked, the item container gets focused instead of the input. So this was to prevent the slot content focus from bubbling up to the outer div. I tried to word it in the pr summary for that

> When the list item is interactive and has content slot, the item can be selected if user clicks outside of the content slot.

Reply:
hmmm I'm not sure we want to do this. Maybe that should be up to the author to prevent bubbling on the input? This doesn't seem right to stop propagation here.

@jcfranco what do you think?

Reply:
What's the use case for the input? This is already essentially a role of checkbox so having an input inside of it might be problematic.

Reply:
Ah, yea the input was an example that the creator for this issue was presenting.

@asangma might be able to speak more about the case in which content slot is used and what happens if content slot is clicked

Reply:
I ended up not using `stopPropagation` and used a styling where container rests on top of the item list so the focus won't bubble up to the current item